Add instruction about push image (#32)
OpenLiberty/guide-openliberty-operator-intro: README.adoc

The `-t` flag in the `docker build` command allows the Docker image to be labeled (tagged) in the `name[:tag]` format. The tag for an image describes the specific image version. If the optional `[:tag]` tag is not specified, the `latest` tag is created by default.

// Static guide instruction
ifndef::cloud-hosted[]

> You need to push your image to a container registry prior to deploying it on any cloud environment. You can skip this step because you are working with a local Kubernetes environment.

Reviewer: ```suggestion
Normally, you must push your image to a container registry before you can deploy it on any cloud environment. You can skip this step because you are working with a local Kubernetes environment.
```
